__tests__/components/AccountMultisigGraph.spec.ts

```diff
@@ -165,26 +142,80 @@ describe('components/AccountMultisigGraph', () => {
+            },
+        };
+        const mockSetAttribute = jest.fn();
+        const mockRemToPixels = jest.fn().mockReturnValue(2);
+        const expectedStyles = `transform: scale(1) translate(2px, 0px); transform-origin: center center;`;
+
+        // Act:
+        const wrapper = getAccountMultisigGraphWrapper(account, stateChanges as AccountState, stubs);
+        const component = wrapper.vm as AccountMultisigGraphTs;
+        const vueTreeChildren = (component.$refs['VueTree'] as Vue).$el.children;
+        vueTreeChildren[0].setAttribute = mockSetAttribute;
+        vueTreeChildren[1].setAttribute = mockSetAttribute;
+        component['remToPixels'] = mockRemToPixels;
+        component['centerGraph']();
+
+        // Assert:
+        expect(mockSetAttribute).toBeCalledTimes(2);
+        expect(mockSetAttribute).nthCalledWith(1, 'style', expectedStyles);
+        expect(mockSetAttribute).nthCalledWith(2, 'style', expectedStyles);
+    });
+
+    test('show graph modal', () => {
+        // Arrange:
+        const account = ({ address: cosig1Address } as unknown) as AccountModel;
+        const stateChanges = {
+            multisigAccountGraph: [],
+        } as Partial<AccountState>;
+        const mockUpdateGraphConfig = jest.fn();
+        const mockCenterGraph = jest.fn();
+        const stubs = {
+            VueTree: {
+                template: '<div class="graph-view"><div/><svg/></div>',
+            },
+        };
+
+        // Act:
+        const wrapper = getAccountMultisigGraphWrapper(account, stateChanges as AccountState, stubs);
+        const graphViewElement = wrapper.find('.graph-view');
+        const component = wrapper.vm as AccountMultisigGraphTs;
+        component['updateGraphConfig'] = mockUpdateGraphConfig;
+        component['centerGraph'] = mockCenterGraph;
+        component['showGraphModal']();
 
         // Assert:
-        expect(component.multisigGraphTree[0].info.accountAddress.address).toBe(msigAddress);
-        expect(component.multisigGraphTree[0].info.cosignatoryAddresses[0].address).toBe(cosig1Address);
-        expect(component.multisigGraphTree[0].info.cosignatoryAddresses[1].address).toBe(cosig2Address);
-        expect(component.multisigGraphTree[0].children[0].info.accountAddress.address).toBe(cosig1Address);
-        expect(component.multisigGraphTree[0].children[1].info.accountAddress.address).toBe(cosig2Address);
-        expect(component.multisigGraphTree[0].children[0].children[0].info.accountAddress.address).toBe(cosig11Address);
-        expect(component.multisigGraphTree[0].children[1].children[0].info.accountAddress.address).toBe(cosig21Address);
-        expect(component.multisigGraphTree[0].children[0].selected).toBe(true);
+        expect(mockUpdateGraphConfig).toBeCalledTimes(1);
+        expect(mockCenterGraph).toBeCalledTimes(1);
+        expect(component.isGraphModalShown).toBe(true);
+        expect(graphViewElement.exists()).toBeTruthy();
```

Comment:
why truthy vs checking explicit value ?

Reply:
Since `.exists()` returns `bool`, yes it makes sense to check explicit value. Thanks, updated.